kyo-prelude/shared/src/main/scala/kyo/Parse.scala

```diff
@@ -167,18 +241,23 @@ object Parse:
             }
         }
 
-    /** Like attempt but commits to the parse result and fails instead of returning Maybe.empty. This is useful when you want to commit to a
-      * parsing branch and prevent backtracking, effectively saying "if we got this far, this must be the correct parse".
+    /** Like attempt but requires the parse to succeed, failing instead of returning Maybe.empty. Use this when a parser must succeed at
+      * this point - if it fails, the entire parse fails with no possibility of backtracking.
+      *
+      * This operation is sometimes known as a "cut" in other parsing libraries, as it cuts off the possibility of backtracking.
       *
       * @param v
-      *   Parser to attempt
+      *   Parser to run
       * @return
       *   Parser result, fails if parser fails with no possibility of backtracking
       */
-    def cut[A: Flat, S](v: A < (Parse & S))(using Frame): A < (Parse & S) =
+    def require[A: Flat, S](v: A < (Parse & S))(using Frame): A < (Parse & S) =
```

Comment:
I had used `cut` because that's what other libraries use but `require` seems more clear.